modules/python/python-logtemplate.c

```diff
@@ -24,34 +24,28 @@
 #include "python-logtemplate-options.h"
 #include "python-logmsg.h"
 #include "python-types.h"
+#include "python-main.h"
 #include "scratch-buffers.h"
 #include "messages.h"
 
 PyTypeObject py_log_template_type;
 PyObject *PyExc_LogTemplate;
 
-void
-py_log_template_free(PyLogTemplate *self)
```

Comment:
This patch contains multiple refactor rounds as well, making it a bit harder to review.